docs/ldml/tr35-keyboards.md

```diff
@@ -1714,9 +1832,13 @@ The following are additions to standard Regex syntax.
     Tooling may choose to suggest an expansion of properties, such as `\p{Mn}` to all non spacing marks for a certain Unicode version.  As well, a set of variables could be constructed in an `import`-able file matching particularly useful Unicode properties.
 
     ```xml
-    <variable id="Mn" value="[\u034F\u0591-\u05AF\u05BD\u05C4\u05C5\…]" /> <!-- 1,985 code points -->
+    <unicodeSet id="Mn" value="[\u034F\u0591-\u05AF\u05BD\u05C4\u05C5\…]" /> <!-- 1,985 code points -->
     ```
 
+- **Ranges**
+
+    To use a range, use a `<unicodeset>` or `<set>` variable.
```

Comment:
I don't think this is correct; we support character classes (which may contain ranges, e.g. `[a-z]`, and is afaik the only use of the term 'range' in regex?).

Reply:
- [ ] `<marquee>TODO</marquee>`